### PR TITLE
NP-978: remove references from the UserAccessRights

### PR DESCRIPTION
**Pull request by axthosarouris**
Reformated the text for better readability

### CODE REVIEW

File: publication-commons/src/test/resources/features/UserAccessRights.feature [hunk 1]
```diff
@@ -70,41 +72,41 @@ Feature: User access rights
...
-    And "theCreator" has given read access to the publication "pubId" to "theUser"
-    When READ is called on behalf of the user "theUser" for the publication "pubID"
-    Then READ returns the publication "pubID"
+  Scenario: users with role USER read unpublished publication that is shared with them
+    Given a publication
+    And the publication's status is DRAFT
+    And the owner of the publication is not the authenticated user
+    And the owner of the publication has given read access to the authenticated user for this publication
+    When READ is called to read the publication on behalf of the authenticated user
+    Then READ returns the saved version of the publication
```
Comment (garshol) [inline publication-commons/src/test/resources/features/UserAccessRights.feature:81]: change "saved" to DRAFT - just to be more explicit ?

File: publication-commons/src/test/resources/features/UserAccessRights.feature [hunk 2]
```diff
@@ -70,41 +72,41 @@ Feature: User access rights
...
-    And the owner of the publication "pubID" is the user "theCreator"
-    And "theCreator" has given write access to the publication "pubId" to "theUser"
-    When UPDATE is called on behalf of the user "theUser" for the publication "pubID"
-    Then UPDATE updates the publication "pubID" stored in "PUBLICATIONS"
-    And UPDATE returns the previously stored version of the publication "pubID"
+  Scenario: users with role USER can update unpublished publications if they have been given write access
+    Given a publication
+    And the publication's status is DRAFT
+    And the owner of the publication is not the authenticated use
+    And the owner fo the publication has given write access to the authenticated user for this publcation
```
Comment (garshol) [inline publication-commons/src/test/resources/features/UserAccessRights.feature:107]: fo -> of

Comment (garshol) [reply publication-commons/src/test/resources/features/UserAccessRights.feature:107]: publcation -> publication

File: publication-commons/src/test/resources/features/UserAccessRights.feature (hunk 1 above)
Comment (axthosarouris) [reply publication-commons/src/test/resources/features/UserAccessRights.feature:81]: "saved" refers to the instance that is in the database

**Pull request merged**